docker-compose.yml

```diff
@@ -1,51 +1,37 @@
-version: '2'
+version: "2"
 services:
+
   db:
     image: postgres:9.5.3
     volumes:
-       - ./pgdata:/pgdata
+      - ./data/pgdata:/var/lib/postgresql/data/pgdata
     environment:
-       PGDATA: /pgdata
-       POSTGRES_USER: listenbrainz
-       POSTGRES_PASSWD: listenbrainz
-       POSTGRES_DB: listenbrainz
+      PGDATA: /var/lib/postgresql/data/pgdata
```

Comment:
They should be there. I added a reference to `common.yml`, but forgot to define them.